cookingWebsite/yourCookbook/viewRecipe.php

```diff
@@ -0,0 +1,112 @@
+<?php
+include('config/init.php');
+verifyUserCooking();
+$recipeId = $_REQUEST['recipeId'];
+$loginId = $_SESSION['loginId'];
+echo yourCookbookEchoHtmlHeader("New Recipe");
+?>
+<script>
+function addTagExistingRecipe(recipeId){
+    var recipeIngredientName = $('#recipeIngredientName').val();
+    var submitItem = $('#submitItem').val();
+    console.log(recipeId, submitItem, recipeIngredientName)
+    $.post('/ajax/saveAddedtag.php', {recipeIngredientName:recipeIngredientName, submitItem:submitItem, recipeId:recipeId}, function(Data){
+          if (Data == "failure"){
+            alert("Error");
+            return false;
+        }
+        // $('.addTag').html(Data)
```

Comment:
Not a big deal, but it might be nice to update the UI to reflect that the recipe has been rated (this could be done with an optimistic write potentially)